src/main/java/dk/cngroup/lentils/service/ImageService.java

```diff
@@ -0,0 +1,114 @@
+package dk.cngroup.lentils.service;
+
+import dk.cngroup.lentils.entity.Hint;
+import dk.cngroup.lentils.entity.Image;
+import dk.cngroup.lentils.entity.formEntity.HintFormObject;
+import dk.cngroup.lentils.exception.ImageFileNotFoundException;
+import dk.cngroup.lentils.exception.ResourceNotFoundException;
+import dk.cngroup.lentils.repository.ImageRepository;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.transaction.Transactional;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Optional;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+@Service
+public class ImageService {
+
+    @Value("${image-directory}")
+    private String imageDirectory;
+
+    private final ImageRepository imageRepository;
+
+    public ImageService(final ImageRepository imageRepository) {
+        this.imageRepository = imageRepository;
+    }
+
+    public Image save(final Image image) {
+        return imageRepository.save(image);
+    }
+
+    public MultipartFile getFile(final String stringPath) {
+        File file = new File(stringPath);
+        Path path = Paths.get(stringPath);
+        String name = file.getName();
+        byte[] content = null;
+        try {
+            content = Files.readAllBytes(path);
+        } catch (final IOException e) {
```

Comment:
Kdyz tam bude vyjimka, co by mela udelat aplikace? Tipuju, ze ignorovani vyjimky neni to spravne reseni, protoze uzivatel se nedozvi, ze doslo k chybe a bude si myslet, ze je vse v poradku.

Reply:
snad jsem to spravila